x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/components/log_rate_analysis.tsx

```diff
@@ -268,6 +268,7 @@ export const LogRateAnalysis: FC<AlertDetailsLogRateAnalysisSectionProps> = ({ r
         </EuiFlexItem>
         <EuiFlexItem>
           <LogRateAnalysisContent
+            source="observability_alert_details"
```

Comment:
We have other alert details pages for other rule types. Could we add a prefix/suffix to specify this is related to Log threshold?

Reply:
Added in bc27b463adabda26003a2e8fc3bbc5b1cf796a62